src/main/java/com/synopsys/integration/jenkins/scan/bridge/BridgeDownloadManager.java

```diff
@@ -1,24 +1,25 @@
 package com.synopsys.integration.jenkins.scan.bridge;
 
+import com.synopsys.integration.jenkins.scan.global.GetOsNameTask;
+import com.synopsys.integration.jenkins.scan.global.Utility;
+import hudson.FilePath;
 import hudson.model.TaskListener;
 
-import java.io.File;
 import java.io.IOException;
-import java.io.InputStream;
 import java.net.*;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.StandardCopyOption;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class BridgeDownloadManager {
     private final TaskListener listener;
+    private final FilePath workspace;
     private final String bridgeBinary = "synopsys-bridge";
+    private final String bridgeBinaryWindows = "synopsys-bridge.exe";
     private final String extensionsDirectory = "extensions";
     private final String versionFile = "versions.txt";
```

Comment:
can be moved to `ApplicationConstants` as these things are also used used in `cleanupOtherFiles()` method